roles/kubernetes-apps/scheduler_plugins/plugins/templates/noderesources/scheduler-config.yml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+leaderElection:
+  leaderElect: {{ enable_leader_election | lower }}
+clientConnection:
+  kubeconfig: /etc/kubernetes/scheduler.conf
+profiles:
+- schedulerName: default-scheduler
+  plugins:
+    score:
+      enabled:
+      - name: NodeResourcesAllocatable
+      disabled:
+      - name: "*"
+  pluginConfig:
+  - name: NodeResourcesAllocatable
+    args:
+      mode: Least
+      resources:
+      - name: cpu
+        weight: 1000000
```

Comment:
why cpu weight is 100000 while memory is 1

Reply:
Good idea, I'll refine it.